### PR TITLE
Added another plugin & modified README, as well as changed RAM for lower-end computers.

### PR DESCRIPTION
**Pull request by CtrlNate**
## Changes
- [x] Changed RAM used from 4GB to 2GB, which should be enough for most servers and is easier on lower-end computers
- [x] Added a new plugin (HolographicDisplays) and documented it in README
- [x] Added crying :'( when things break
- [x] Fixed documentation (1.12 --> 1.12.2)
- [x] FIxed typo on line 72 of server-essentials
- [x] Added more lines to the ending thingy, saying to add an issue to request a plugin to be added
- [x] You should add me as a contributor thing so I can update the version as new versions of Minecraft come out

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
-* ProtocolLib
-* Multiverse (Core and NetherPortals)
-* PlugMan
-* PermissionsEx
-* Vault
-* WorldEdit
-* WorldGuard
-
-It also creates a simple start script that runs your server with *4GB of RAM*, and automatically sets the EULA to agreed. (*Note: by using this script __you are agreeing to the Mojang EULA__*)
+* Spigot(http://spigotmc.org)
```
Comment (foxfirecodes) [inline README.md]: Please use proper markdown link format, e.g. `[text](url)`

File: server-essentials [hunk 2]
```diff
@@ -135,6 +136,14 @@ task "Cleaning up"; clean; taskdone
 
 
 
+echo ""
```
Comment (foxfirecodes) [inline server-essentials]: You don't need quotes for an empty echo.

File: server-essentials [hunk 3]
```diff
@@ -135,6 +136,14 @@ task "Cleaning up"; clean; taskdone
...
+echo "$(tput bold)Welcome to server-essentials!"
+echo "Written by Rayzr522 & (a little but) ACA30"
```
Comment (foxfirecodes) [inline server-essentials]: Just remove `(a little but)`, `Rayzr522 & ACA30` is fine :P

File: server-essentials [hunk 4]
```diff
@@ -135,6 +136,14 @@ task "Cleaning up"; clean; taskdone
...
+echo ""
 echo "$(tput smul)$(tput setaf 2)$(tput bold)
 Server setup complete. You can run the server with ./start.sh
 $(tput rmul)$(tput sgr0)"
+echo ""
+echo "If you would like a plugin added,"
+echo "please create an issue on GitHub"
```
Comment (foxfirecodes) [inline server-essentials]: This can all be one line.

File: server-essentials (hunk 3 above)
Comment (foxfirecodes) [inline server-essentials]: Please move this and the "if you want a plugin added, create an issue" to before all the installation of the plugins and the setup of the server.

**Review by foxfirecodes — CHANGES_REQUESTED** [review]
Just a few quick changes and we'll be good to go!

File: README.md (hunk 1 above)
Comment (CtrlNate) [reply README.md]: (Not gonna lie you were the one to put those there but sure okay)

**Pull request merged**